contracts/lib/GatewayLib.sol

```diff
@@ -105,8 +79,15 @@ library GatewayLib {
          * Verify the remote OpenST contract against the committed state
```

Comment:
This should be just a comment and not documentation comment. remove a `*` from line 78